composer.json

```diff
@@ -76,7 +77,11 @@
         "monolog/monolog": "~1.11",
         "ircmaxell/password-compat": "~1.0",
         "ocramius/proxy-manager": "~0.4|~1.0",
-        "egulias/email-validator": "~1.2"
+        "egulias/email-validator": "~1.2",
+        "phpdocumentor/reflection": "~1.0"
```

Comment:
Oh, I've just found the other composer.json with the discussion about this topic. I'll join there. Nevermind. ;-)